### DISCUSSION TITLE
[BP#67] Suppression de la pratique « Remplacer les $i++ par ++$i »

### DISCUSSION THREAD

**Discussion by rlemaire** [cnumr/best-practices#19] [Bonnes pratiques] [answered]
On économise seulement une instruction processeur, ce qui est loin de l'impact de la plupart des pratiques et de l'enjeu en général.

**Comment by arnolem** [comment]
En complément, quelques tests réalisés sur un PHP 8
 - https://twitter.com/SIPOnline/status/1445080597152534539
 - https://twitter.com/nicolaslegland/status/1444933982420881408

**Comment by ACTLEM** [comment, marked as answer]
https://github.com/cnumr/best-practices/issues/123
